src/jdk.jfr/share/conf/jfr/default.jfc

```diff
@@ -734,6 +734,18 @@
       <setting name="threshold" control="file-threshold">20 ms</setting>
     </event>
 
+    <event name="jdk.SocketConnect">
+      <setting name="enabled">true</setting>
+      <setting name="stackTrace">true</setting>
+      <setting name="threshold" control="socket-threshold">20 ms</setting>
+    </event>
+
+    <event name="jdk.SocketConnectFailed">
+      <setting name="enabled">true</setting>
+      <setting name="stackTrace">true</setting>
+      <setting name="threshold" control="socket-threshold">20 ms</setting>
```

Comment:
I think it needs to be a duration event, otherwise it's possible to a floor events when connections fail immediately. With the current proposal then it is at least configurable.